src/app.js

```diff
@@ -43,29 +43,36 @@ app.get('/help', (req, res) => {
 })
 
 app.get('/weather', (req, res) => {
-    if (!req.query.address) {
+    if (!req.query.address && !req.query.lat && !req.query.lon) {
```

Comment:
https://javascript.info/destructuring-assignment#object-destructuring

Reply:
I don't quite understand how to implement destructuring here, could you please clarify?
Also please chec kcurrent implementation.

Reply:
```javascript
const {address, lat, lon } = req.query
```

Reply:
Fixed. (I guess)